### PR TITLE
Refactor set version to template

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Refactor set version to template

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/docs/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

**Comment by TravisEz13** [comment]
FYI, CI is not relevant to this PR

**Review by rjmholt — APPROVED** [review]

File: .vsts-ci/templatesGeneric/setVersionStepsTemplate.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      }
+      $namespace = $releaseTag.ToLowerInvariant()
+      Write-Host "##vso[task.setvariable variable=dockerNamespace;]$namespace"
+      $version = $releaseTag -replace '^v', ''
+      $command = "vso[task.setvariable variable=version]$version"
+      Write-Host $command
+      Write-Host "##$command"
+    displayName: 'Set Version'
+    name: get
+    condition: and( succeededOrFailed(), ne(variables['Channel'],''))
```
[the remark this answers is not in the document]
Comment (TravisEz13) [reply .vsts-ci/templatesGeneric/setVersionStepsTemplate.yml:28]: doesn't include canceled

[the remark this answers is not in the document]
Comment (TravisEz13) [reply .vsts-ci/templatesGeneric/setVersionStepsTemplate.yml:28]: `always()` is the condition to run always

**Comment by TravisEz13** [comment]
feel free to give feedback, but I merged so that 484 could use this code.

**Pull request merged**